src/daniil_efremov_ex/ex02/message_entropy.py

```diff
@@ -0,0 +1,34 @@
+import math
+
+
+def letter_freq(message):
+    lower_txt = message.lower()
+    single_item = [item for item in lower_txt]
+
+    return dict((item, single_item.count(item)) for item in single_item)
+
+
+def entropy(message):
+    number_of_letters = len(message)
+
+    count_dict = letter_freq(message)
+    occurences = dict()
+    frequency = dict()
+    entro = 0
+
+    for code in range(256):
+
+        if chr(code) in count_dict:
```

Comment:
This works, but there's a easier way to do this. Maybe you want to check the suggested solution.